HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -758,6 +758,18 @@ def self.apply_appliances(hpxml, nbeds, eri_version)
       if clothes_dryer.usage_multiplier.nil?
         clothes_dryer.usage_multiplier = 1.0
       end
+      if clothes_dryer.rated_flow_rate.nil?
+        if not clothes_dryer.combined_energy_factor.nil?
+          if clothes_dryer.combined_energy_factor < 4.5
+            clothes_dryer.rated_flow_rate = 100.0
+          end
+        end
+        if not clothes_dryer.energy_factor.nil?
+          if clothes_dryer.energy_factor < 5.175
+            clothes_dryer.rated_flow_rate = 100.0
+          end
+        end
```

Comment:
Where did this come from?

This will need to be updated to use the IsVented input.

Reply:
HSP.

Reply:
Sorry, I understand 100 cfm comes from the HSP but where does the dependency on efficiency come from? A heat pump clothes dryer, which has a very high efficiency, can be either vented or unvented for example.